src/main/drivers/bus_spi.c

```diff
@@ -499,6 +499,16 @@ FAST_CODE void spiProcessSegmentsDMA(const extDevice_t *dev)
     spiInternalStartDMA(dev);
 }
 
+static void spiPreInitStream(const extDevice_t *dev)
+{
+    // Prepare the init structure for the next segment to reduce inter-segment interval
+    // (if it's a "buffers" segment, not a "link" segment).
+    busSegment_t *segment = (busSegment_t *)dev->bus->curSegment + 1;
+    if (segment->len > 0) {
+        spiInternalInitStream(dev, segment);
```

Comment:
```suggestion
    busSegment_t *next= (busSegment_t *)dev->bus->curSegment + 1;
    if (next->len > 0) {
        spiInternalInitStream(dev, next);
```
(or `nextSegment` but code is only 4 lines)